staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -126,6 +135,23 @@ type Config struct {
 	// FlowControl, if not nil, gives priority and fairness to request handling
 	FlowControl utilflowcontrol.Interface
 
+	// PeerProxy, if not nil, sets proxy transport between kube-apiserver peers for requests
```

Comment:
Do these fields and definitions belong in the generic apiserver library? I expected them in kube-apiserver or kube-aggregator. A generic apiserver wouldn’t have a proxy certificate to dictate user identity to arbitrary other servers, right?

Reply:
Can you specify which file exactly you mean if I am to make these additions in the kube-apiserver/kube-aagregator ?

Reply:
Included changes from the commit shared above.